src/TimePicker/TimePicker.js

```diff
@@ -30,6 +30,11 @@ const TimePicker = React.createClass({
     defaultTime: React.PropTypes.object,
 
     /**
+     * Disables the DatePicker.
+     */
+    disabled: React.PropTypes.bool,
+
```

Comment:
'If true, the TimePicker is disabled.' 
https://github.com/callemall/material-ui/wiki/Documentation#description-wording)

DatePicker isn't currently not following that convention, so not your fault.